torch/csrc/jit/codegen/cuda/kernel_expr_evaluator.cpp

```diff
@@ -34,14 +197,12 @@ c10::optional<Int::ScalarType> ExpressionEvaluator::evaluate(const Val* value) {
     return value->as<Int>()->value();
   }
 
-  // Is the value known (either explicit binding or memoized)?
+  // Is the value known (either explicit binding)?
   const auto pre_eval_it = known_values_.find(value);
   if (pre_eval_it != known_values_.end()) {
     return pre_eval_it->second;
   }
-
   value->accept(this);
-
   const auto post_eval_it = known_values_.find(value);
   return post_eval_it != known_values_.end()
       ? c10::optional<Int::ScalarType>(post_eval_it->second)
```

Comment:
Why did you modify this function? As far as I understand, the use of the new expression evaluator shouldn't affect this interface, right?

Reply:
Just me being sloppy removing some additional asserts I had for development purposes.